rust-toolchain.toml

```diff
@@ -1,2 +1,5 @@
 [toolchain]
+channel = "stable"
```

Comment:
I think this is fine: any concerns @nicholasbishop ?

Reply:
Seems fine to me, but just for my own edification, why is this part of the change necessary?

Reply:
So that `rust-bin.fromRustupToolchainFile ../rust-toolchain.toml` can pick the right toolchain for Nix.